starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Apple Pie Recipe</title>
+   <meta charset="utf-8">
+   <title>Apple Pie Recipe</title>
+   <link rel="stylesheet" href="./css/main.css"/>
 </head>
+
 <body>
-Apple Pies
+<h1> 
+    <div class="container">
+        <img src="./images/apple-pie.jpg" alt="Apple Pie" style="width:85%; height : 500px">
+        <div class="text-block"> 
+          <h4>Apple Pie</h4>
+        </div>
+      </div>
+</h1>
+<div id="description" 
+<p>
 This was my grandmother's apple pie recipe. I have never seen another one
 quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+prizes in local competitions. I hope it becomes one of your favorites as well</p>
+<img src="./images/cooking-info.png">
+</div>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+<div class="ingredients">
+<h2>Ingredients</h2>
+<ul>
```

Comment:
Here the structure of your code is good, but it's a bit too "flat". Indenting is very important as it gives you a clear view of what is nested inside what, thus allowing you to quickly check if you organized your code well, and also helps other people to understand it!
(your code is probably going to be looked at a lot by other developers so you definitely want to keep it tidy)